package.json

```diff
@@ -26,7 +26,7 @@
   },
   "name": "root",
   "scripts": {
-    "dev:gui": "corepack pnpm run --parallel -r --filter enso-gui --filter ydoc-server-nodejs  --aggregate-output /^^^^dev:/",
+    "dev:gui": "corepack pnpm run --parallel -r --filter enso-gui --filter ydoc-server-nodejs --aggregate-output /^^^^dev:/",
```

Comment:
Because enso-gui now depends on the project-manager-shim package, the command `pnpm run dev:gui` now requires to compile the project-manager-shim package first with `pnpm run compile`. 

I'm not sure why because enso-gui already depends on enso-common, and the `dev:gui` command worked somehow without the need to compile enso-common. I'd appreciate your thoughts on how to fix that.

Reply:
> I'm not sure why because enso-gui already depends on enso-common, and the dev:gui command worked somehow without the need to compile enso-common. I'd appreciate your thoughts on how to fix that.

AFAIK the enso-common also needs to be compiled (perhaps you have some leftovers? `./run ide build` does compile for example). And this is to be improved once bazel lands properly.